index.js

```diff
@@ -1,26 +1,48 @@
 // To do:
-// Find an international weather app - DONE
+// Find an international weather API - DONE
 // Read documentation and create API call - DONE
 // Add basic forecast to index.html - DONE
-// Figure out how to select a forecast by location (read https://openweathermap.org/api/geocoding-api)
+// Figure out how to select a forecast by location (read https://openweathermap.org/api/geocoding-api) - DONE
 // Create a form for user to select their chosen location's forecast
+// Figure out how to let user see and choose options for their city (e.g. which 'London' do they mean)?
 // Use flexbox and CSS to improve page layout
 // Add dynamic page background, selected by keywords in shortForecast
 
+function getLocation() {
+  // TO DO: move city name to a parameter of this function
+  return (
+    fetch('http://api.openweathermap.org/geo/1.0/direct?q=London&limit=5&appid=886b4d9fa52f373586e1ac5a100c5cf1')
+      .then(response => response.json())
+      .then(json => {
+        const locations = json;
+        console.log('Locations: ', locations);
+        // return the lat and lon of the first location in locations array
+        const latLon = [locations[0].lat, locations[0].lon];
+        return latLon;
+      })
+      .then(latLon => {
+        const [lat, lon] = latLon;
+        return getForecast(lat, lon);
+      })
+      .catch((error) => console.error(error))
+      );
+}
 
-function getForecast() {
-  fetch('https://api.openweathermap.org/data/2.5/weather?q=London,uk&units=metric&APPID=886b4d9fa52f373586e1ac5a100c5cf1')
+function getForecast(lat, lon) {
+  return fetch(`https://api.openweathermap.org/data/2.5/weather?lat=${lat}&lon=${lon}&units=metric&APPID=886b4d9fa52f373586e1ac5a100c5cf1`)
     .then(response => response.json())
     .then(json => {
+      return json;
+    })
+    .catch((error) => console.error(error))
```

Comment:
Something I thought of just now. I wonder if having the .catch() clause here is unnecessary since you have one on the getLocation() call inside of main(). There may be a scenario where you don't want to intercept errors at a lower level. I would call this kind of error handling an intermediate concept. Mostly I wanted to share my thought and show a review comment. 👍